nestauk/asf_heat_pump_affordability
# 18 investigate effect of room number on pre- and post-1950 installation costs

Fixes #18 

# Description

- Investigate effect of room number on pre- and post-1950 installation costs within the different archetypes.
- Identify that room number explains some of the difference in pre- and post-1950 installation costs within all archetypes except detached houses.
- Create a new dataset of installation cost estimates at each decile 1-9 for each archetype at the median number of rooms for that property type.

# Instructions for Reviewer

In order to test the code in this PR you need to:
- run `jupytext --to notebook asf_heat_pump_affordability/notebooks/investigate_effect_of_room_number.py`
- run the notebook

Please could you:
- review for general errors
- double check data preprocessing is appropriate
- double check reasoning for setting alpha in model.summary() to sqrt 0.05
- double check logic to calculate difference in pre- and post-1950 coefficients and confidence intervals

# Checklist:
